node/network/statement-distribution/src/lib.rs

```diff
@@ -1500,53 +1547,62 @@ impl StatementDistribution {
 	async fn run(
 		self,
 		mut ctx: impl SubsystemContext<Message = StatementDistributionMessage>,
-	) -> SubsystemResult<()> {
+	) -> std::result::Result<(), Fatal> {
 		let mut peers: HashMap<PeerId, PeerData> = HashMap::new();
+		let mut authorities: HashMap<AuthorityDiscoveryId, PeerId> = HashMap::new();
 		let mut active_heads: HashMap<Hash, ActiveHeadData> = HashMap::new();
+
+		let mut runtime = Runtime::new(self.keystore.clone());
```

Comment:
FWIW, I really don't like that this is called 'Runtime' since that has a separate meaning and this is really just a session info cache + keystore thing

Reply:
I'll go with `RuntimeInfo` then, as it is meant as a convenient and efficient API for accessing runtime information. (E.g. also core status and such)